src/pages/configeditor/overview.md

```diff
@@ -27,6 +27,7 @@ In this document, find instructions for:
 
 - Installing Configuration Editor [Using Docker](/configeditor/download_docker/)
 - Installing Configuration Editor [On Red Hat OpenShift](/configeditor/download_openshift)
+- [Migrating](/configeditor/migrating_data) exisitng Early Access Program artifacts
```

Comment:
typo here:  should be "existing".
More of a question for my understanding:  Is "Early Access Program artifacts" how they are known to our users?